src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  padding: 1em 0;
+}
+
+h1,
+h2,
+p,
+ol {
+  margin: 0;
+}
```

Comment:
You should style using classes here too. Add classes to your headers, paragraphs etc.